### PR TITLE
Adhere to oauth2 spec

### PR DESCRIPTION
**Pull request by junnae**
Updating access-token-api as well

### CODE REVIEW

**Review by adriantimberlid — COMMENTED** [review]
LGTM

File: openapi/access-token/server.yaml [hunk 1]
```diff
@@ -17,7 +17,6 @@ paths:
         - Client Authorization Service
       summary: Authenticate with the BankAxept ePayment system and receive an Authorization token.
       requestBody:
-        required: false
```
Comment (junnae) [inline openapi/access-token/server.yaml:20]: this defaults to false. So ideally we would set this to required, and also require the "grantType". But it doesn't work with our API, and I think it's related to spring webflux not being able to take in these kinds of parameters directly, you have to get them through `serverwebexchange getformdata`

File: java/pom.xml [hunk 2]
```diff
@@ -83,7 +83,7 @@
         </dependencies>
     </dependencyManagement>
     <properties>
-        <swagger.parser.version>2.0.30</swagger.parser.version>
+        <swagger.parser.version>2.1.6</swagger.parser.version>
```
Comment (junnae) [inline java/pom.xml:86]: https://mvnrepository.com/artifact/org.openapitools/openapi-generator/6.2.1

**Review by thxmasj — APPROVED** [review]
⚡

**Pull request merged**